### ISSUE TITLE
Add notes to syllabus _post directory for 9/24

### ISSUE THREAD

**Issue by davclark** [hackingmeasurement/hackingmeasurement.github.io#50]
There is markdown [here](https://etherpad.mozilla.org/HMclass-launch). Please take that content and create an appropriately formatted post in this repo under `syllabus/_posts`. You can copy over _template.md and be sure to format the filename to include the date as the beginning of the filename!

**Comment by davclark** [comment]
Hi @yangraymond, time is passing. The idea is for notes to go up pretty quickly. Are you going to be able to get this done before this Thursday? If not, I'll do it, and you'll have to find another way to fulfill your note-taking requirement.

**Comment by yangraymond** [comment]
Yes I can do this, I wasn't aware this was still open. Getting this done asap.

**Comment by davclark** [comment]
No worries! Glad the issue tracker is working ;)

**Comment by davclark** [comment]
@yangraymond I checked and realized that the link above was not correct. Also, you have not committed to the repository. When you marked this as "closed" what did you mean by that?

**Comment by davclark** [comment]
Ack - sorry, I think that link _was_ correct, and I see that you did do this, but you're not "getting credit". Probably this is because you have not confirmed your email with github or similar. See here, and you'll note that the commit you made doesn't have a clickable username:

https://github.com/BIDS-collaborative/hackingmeasurement.github.io/commits/gh-pages

Can you check that your git account is set up correctly?
